promotion/src/main/java/org/cryptimeleon/incentive/promotion/model/Basket.java

```diff
@@ -0,0 +1,21 @@
+package org.cryptimeleon.incentive.promotion.model;
+
+import org.cryptimeleon.math.structures.rings.zn.Zn;
+
+import java.math.BigInteger;
+import java.util.List;
+import java.util.UUID;
+
+public class Basket {
+    public final UUID basketId;
```

Comment:
same question here, role of this class?
isn't this basket server stuff?

Reply:
We need this in order to model promotions. I am not sure if sharing this logic would be a good idea